split-webapp/split/src/pages/Login.js

```diff
@@ -23,6 +23,16 @@ class Login extends Component {
     };
   }
 
+  componentDidMount = () => {
+    const inputField = document.getElementById("inner-container-id");
+    inputField.addEventListener("keyup", function(event) {
```

Comment:
Would using the `onKeyUp` prop on the `<Box>` be clearer and safer (similar to the other event handlers)? Or was there a reason why the keyup handler was registered inside `componentDidMount`?

Reply:
Changes made to use onKeyUp prop and removed componentDidMount

Reply:
Sweet! Looking forward to your changes when pushed.